packages/react-native-reanimated/Common/cpp/reanimated/Fabric/ReanimatedCommitHook.cpp

```diff
@@ -17,10 +17,13 @@ namespace reanimated {
 ReanimatedCommitHook::ReanimatedCommitHook(
     const std::shared_ptr<UIManager> &uiManager,
     const std::shared_ptr<UpdatesRegistryManager> &updatesRegistryManager,
-    const std::shared_ptr<LayoutAnimationsProxy> &layoutAnimationsProxy)
+    const std::shared_ptr<LayoutAnimationsProxy_Legacy> &layoutAnimationsProxyLegacy,
+    const std::shared_ptr<reanimated_experimental::LayoutAnimationsProxy_Experimental>
```

Comment:
Why do we need `LayoutAnimationsProxy_Experimental` to be in `reanimated_experimental` namespace instead of just `reanimated`?